src/commands/install.yml

```diff
@@ -88,6 +88,30 @@ steps:
                   exit 1
               esac
 
+              # Install `less` for paging if not present
+
+              if [ -z "$(command -v less)" ]; then
+                case $SYS_ENV_PLATFORM in
+                  linux)
+                    if [ -x "$(command -v apt-get)" ]; then
+                      $SUDO apt-get install less
+                    else
```

Comment:
Alpine will have `less` installed in any case via `busybox` so checking for `apk` is probably pointless. Are there other major package managers this should check for? Or is there even a nicer way of foing that check?

Reply:
I think supporting `apt` is sufficient for now